src/cat.sol

```diff
@@ -56,15 +56,17 @@ contract Cat is LibNote {
     // --- Data ---
     struct Ilk {
         address flip;  // Liquidator
-        uint256 chop;  // Liquidation Penalty   [ray]
-        uint256 lump;  // Liquidation Quantity  [wad]
+        uint256 chop;  // Liquidation Penalty  [ray]
+        uint256 lump;  // Liquidation Quantity [rad]
     }
 
     mapping (bytes32 => Ilk) public ilks;
 
-    uint256 public live;  // Active Flag
-    VatLike public vat;   // CDP Engine
-    VowLike public vow;   // Debt Engine
+    uint256 public box;    // Max Dai out for liquidation        [rad]
```

Comment:
You might want to preserve the original storage slot ordering. It was just a coincidence that it was alphabetical order too.

Reply:
good point, tooling could be depending on that.